profiles/common/modules/custom/multisite_drupal_toolbox/multisite_drupal_toolbox.module

```diff
@@ -807,28 +807,13 @@ function _disable_block($module, $delta) {
  *   Operation to perform, either 'add' or 'delete.
  */
 function multisite_drupal_toolbox_config_solr_bundle($bundle, $op) {
-  if (!module_exists('apachesolr')) {
-    watchdog(
-      'multisite_drupal_toolbox',
-      'Function %function was called with bundle %bundle and op %op, but the required apachesolr module is not enabled. You either need to add the module apachesolr as a dependency to the feature you are calling this function from, or prevent the call from happening if apachesolr is not a strict dependency and is disabled.',
-      array(
-        '%function' => __FUNCTION__,
-        '%bundle' => $bundle,
-        '%op' => $op,
-      ),
-      WATCHDOG_ERROR
-    );
-
+  if (!in_array($op, array('delete', 'add'))) {
     return;
   }
 
-  if ($op != 'delete' && $op != 'add') {
+  if (!module_exists('apachesolr')) {
     return;
   }
-  else {
-    $info = new stdClass();
-    $info->type = $bundle;
-  }
 
   // Update hook cache to find new hooks (see module_implements()).
   cache_set('module_implements', array(), 'cache_bootstrap');
```

Comment:
Apparently it's done multiple time in the module and [once in Drupal core](http://cgit.drupalcode.org/drupal/tree/includes/module.inc?h=7.54#n741).